vmc/resource_vmc_cluster.go

```diff
@@ -165,15 +165,25 @@ func resourceClusterRead(d *schema.ResourceData, m interface{}) error {
 		d.SetId("")
 		return nil
 	}
+	clusterExists := false
 
+	for _, clusterConfig := range sddc.ResourceConfig.Clusters {
+		if strings.Contains(clusterConfig.ClusterId, clusterID) {
+			clusterExists = true
+		}
+	}
+	if !clusterExists {
+		log.Printf("Unable to retrieve cluster with ID %s", clusterID)
```

Comment:
does this log statement work?

Reply:
Yes. I have tested it. You need to export TF_LOG=TRACE. You can even write the logs to a file by exporting TF_LOG_PATH=terraform.log (for example).

Reply:
Done